### DISCUSSION TITLE
RARE DEATH SCREEN IN CODENAME?????????????????????????

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#434] [Show and tell]
This is the first and only time I've ever gotten this screen, and of course it's not in vslice
![image](https://github.com/user-attachments/assets/49948451-26c9-454b-bef0-3e4b6fe3aec6)

**Comment by NeeEoo** [comment]
Thats the pause menu

**Reply by THEkatinamicrowave** [reply]
what I thought this was pause menu
![image](https://github.com/user-attachments/assets/218e7028-746f-43db-a396-a89ee23aff24)

**Comment by NexIsDumb** [comment]
thats a rare screen of the pause menu called "gitaroo pause" which was also in base game, you can also disable it on scripts and it has a chance of popping up like 1/1000 iirc
